SanUI/Modules/AddonMods/coolline.lua

```diff
@@ -34,11 +34,11 @@ function S.modCoolLine(event)
 			CoolLine:ClearAllPoints()
 			if TukuiStanceBar and NumForms > 0 then 
 				CoolLine:SetPoint("BOTTOMLEFT",TukuiStanceBar,"BOTTOMRIGHT",spacing,spacing)
-				CoolLine:SetPoint("TOPLEFT",TukuiStanceBar,"TOPRIGHT",spacing,-spacing)
```

Comment:
Why are you removing this?

Reply:
We don't need a TOPLEFT anchor if we have a TOPRIGHT anchor on action bar 3 just a few lines below...

https://github.com/SanUI/SanUI/blob/100ef0af3b2dcf038d2272250b1a87abcfd2ad9d/SanUI/Modules/AddonMods/coolline.lua#L39

Reply:
Wait, am I missing something? That's a BOTTOMRIGHT you're referring to, isn't it? Line 41, right?